hybrid/decomposers.py

```diff
@@ -828,6 +940,25 @@ def _make_cubic_lattice(dimensions):
                                         ])
     return cubic_lattice_graph
 
+def _make_kings_lattice(dimensions, is_open=(True,True)):
```

Comment:
Can we add type annotations here as well?

Reply:
My impression was, based on pattern recognition only, that there is a convention where typing could be omitted from hidden functions. I guess its better to have it everywhere new functionality is added?

Reply:
There's no strict requirement for type annotations, but we try to annotate all new code we add.